controller/rest/crdvalidatewebhook.go

```diff
@@ -347,7 +356,9 @@ func (q *tCrdRequestsMgr) crdQueueProc() {
 					if retryCount > 3 {
 						log.Printf("Crd dequeu proc Plicy lock FAILED")
 						// push req back to kv queue
-						q.crdProcEnqueue(record) // record is *admissionv1beta1.AdmissionReview
+						if _, err := q.crdProcEnqueue(record); err != nil { // record is *admissionv1beta1.AdmissionReview
+							log.Printf("crdProcEnqueue error, %s", err.Error())
```

Comment:
I wonder what Is the reason why `Printf` is used in this particular file.  Can we use `Infof()` instead?

Reply:
changed.